release_dashboard/test/fakes/fake_start_context.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+import 'package:platform/platform.dart';
+
+import 'fake_process_manager.dart';
+import 'fake_stdio.dart';
+
+const String kFlutterRoot = '/flutter';
+const String kCheckoutsParentDirectory = '$kFlutterRoot/dev/tools/';
+
+/// Initializes a fake [StartContext] in a fake local environment.
+class FakeStartContext extends StartContext {
+  factory FakeStartContext({
+    String candidateBranch = 'flutter-1.2-candidate.3',
+    Checkouts? checkouts,
```

Comment:
this doesn't need to be a parameter, right? we're just overriding it on line 46.

Reply:
Added null-aware operators `??=`